sound/soc/sof/Makefile

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
 
 snd-sof-objs := core.o ops.o loader.o ipc.o pcm.o pm.o debug.o topology.o\
-		control.o trace.o utils.o sof-audio.o
+		control.o trace.o utils.o sof-audio.o ipc4.o
```

Comment:
do we need both ipc and ipc4 in the same module?

Reply:
> do we need both ipc and ipc4 in the same module?

Yes, there are still some common functions in ipc.c. For example, snd_sof_ipc_init and snd_sof_ipc_free.